assets/lib/share-link/share-link.js

```diff
@@ -1,10 +1,10 @@
 /**
  * By Elementor Team
  */
-( function( $ ) {
-	window.ShareLink = function( element, userSettings ) {
-		var $element,
-			settings = {};
+( function() {
+	window.ShareLink = function( eElement, userSettings ) {
```

Comment:
Is `eElement` used in the code?

Reply:
hmm, I don't see it, but is doesn't make sense, there should be.